eng/pipelines/ci.yml

```diff
@@ -73,15 +73,4 @@ extends:
               - template: /eng/pipelines/jobs/website.yml
 
           - template: /eng/pipelines/jobs/e2e-job.yml@self
-          - job: AutorestChecks
```

Comment:
I believe that this is still used by management plane sdks, especially for updates to existing specs.